src/common/InfoText.js

```diff
@@ -0,0 +1,84 @@
+// Front Page:
+export const frontPage = 'This application aims to discuss the inefficiencies in the storage and retrieval of medical record data. \
```

Comment:
FIX: These strings shouldn't be variables. Most things on a website are strings, so it's much easier to place the strings where they should be. The only alternative is if there are multiple questions/answers all in the same format, then placing them in a JS object/array or a JSON file would make mapping them to a React component or fragment easier.

Reply:
Sounds good, I'll move the strings into the code. I will leave this file though, as some of the strings have yet to be used, so we want to keep the content for later.

Reply:
Actually we decided to keep it for now, we worked like this because Kevin was making/adjusting the text while I was coding the boxes, and since we will continue doing that, it's easier for now. Am happy to move these though down the road, just wanna get this merged in during our in class meeting